coil-core/src/commonMain/kotlin/coil3/request/imageRequests.kt

```diff
@@ -30,6 +30,15 @@ expect val ImageRequest.crossfadeMillis: Int
 
 internal const val DEFAULT_CROSSFADE_MILLIS = 200
 
+fun ImageRequest.Builder.crossfadeBetweenImages(enable: Boolean) = apply {
```

Comment:
Can you add a short doc comment for this method?

Also we should add `ImageLoader.Builder.crossfadeBetweenImages` so this can be configured per-`ImageLoader`.